build.xml

```diff
@@ -13,7 +13,7 @@
 	<property name="installer" value="org/rsna/installer"/>
 	<property name="dicomanonymizertool" value="org/rsna/dicomanonymizertool"/>
 
-	<property name="ctp" value="/Development/CTP"/>
```

Comment:
In order to build this module independently, I had to remove the hard-coded parent folder name. However, the build still expects CTP project available as a folder within parent folder.